src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.FruitShopStorage;
+import core.basesyntax.model.Fruit;
+import java.util.List;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StorageDaoImplTest {
+    private static StorageDao daoTest;
+
+    @BeforeClass
+    public static void setUp() {
+        daoTest = new StorageDaoImpl();
+    }
+
+    @After
+    public void clearStorage() {
+        FruitShopStorage.storageFruits.clear();
+    }
+
+    @Test
+    public void add_fruit_Ok() {
+        Fruit apple = new Fruit("apple", 20);
+        daoTest.addFruit(apple);
+        boolean actual = FruitShopStorage.storageFruits.size() == 1;
+        assertTrue(actual);
+        assertTrue(FruitShopStorage.storageFruits.contains(apple));
+    }
+
+    @Test
+    public void get_fruit_Ok() {
+        Fruit apple = new Fruit("apple", 20);
+        daoTest.addFruit(apple);
+        String expected = "apple";
+        String actual = daoTest.getFruit("apple").getName();
+        assertEquals(expected, actual);
```

Comment:
Put the object in the storedge and retrieve it with Dao, and compare the two retrieved objects